### DISCUSSION TITLE
Best way to clear disk space?

### DISCUSSION THREAD

**Discussion by eskrano** [filecoin-project/lotus#7680] [General Q&A] [answered]
Hello, im using lotus wallet on my VPS and its took 160+GB ssd for now, what the best way to clear space and keep it used like 100GB for chain data store? 

Thanks,

**Comment by rjan90** [comment, marked as answer]
Hey 😄 You can prune the chain by following this guide: [https://docs.filecoin.io/get-started/lotus/chain/#compacting-the-chain-data](https://docs.filecoin.io/get-started/lotus/chain/#compacting-the-chain-data)

**Comment by eskrano** [comment]
@rjan90 yes its works i think but i do not have any free space on the server right now and i can not do anything, can i delete part of all chain data?

**Comment by rjan90** [comment]
@eskrano Then I would stop the daemon, remove the content in `~/.lotus/datastore/chain`, and download a snapshot. It will cause some downtime, but then you will start with a "pruned" chain again.

**Comment by de-authority** [comment]
To be honest. 160Gb is really not enough for running a lotus node. you'll have to prune the chain constantly. if so, you should write a script to do this job.
